py3status/modules/air_quality.py

```diff
@@ -0,0 +1,190 @@
+countries have their own air quality indices, corresponding to different national
+air quality standards.
+
+Configuration parameters:
+    auth_token: Required. Personal token from http://aqicn.org. (default 'demo')
+    cache_timeout: refresh interval for this module. A message from the site:
+        The default quota is max 1000 (one thousand) requests per minute (~16RPS)
+        and with burst up to 60 requests. See http://aqicn.org/api/
+        (default 3600)
+    color_good: Good (default '#009966')
+    color_hazardous: Hazardous  (default '#7E0023')
+    color_moderate: Moderate (default '#FFDE33')
+    color_unhealthy: Unhealthy (default '#CC0033')
+    color_unhealthy_sensitive: Unhealthy for Sensitive Groups (default '#FF9933')
+    color_unknown: Unknown (default '#FFFFFF')
+    color_very_unhealthy: Very Unhealthy (default '#660099')
+    format: display format for this module (default 'AQI: {aqi} {category}')
+    icon_good: (default 'Good')
+    icon_hazardous: (default 'Hazardous')
+    icon_moderate: (default 'Moderate')
+    icon_unhealthy: (default 'Unhealthy')
+    icon_unhealthy_sensitive: (default 'Unhealthy for Sensitive Groups')
+    icon_unknown: (default 'Unknown')
+    icon_very_unhealthy: (default 'Very Unhealthy')
+    location: location name/uid to query. To search for nearby stations in
+        Kraków, use `curl http://api.waqi.info/search/?token=YOUR_TOKEN&keyword=kraków`
+        We recommend you to use uid instead of name in location, eg "@8691"
+        (default 'Shanghai')
+
+Format placeholders:
+    {aqi} air quality index
+    {category} health risk category
+    {location} location name/uid
+    {random} output from "random" parameter in `iaqi` array.
+
+__Note: Station may have their set of custom parameters such as {pm25}, {pm10}, etc.__
+
+Color options:
+    color_good: Good (default '#009966')
+    color_hazardous: Hazardous  (default '#7E0023')
+    color_moderate: Moderate (default '#FFDE33')
+    color_unhealthy: Unhealthy (default '#CC0033')
+    color_unhealthy_sensitive: Unhealthy for Sensitive Groups (default '#FF9933')
+    color_unknown: Unknown (default '#FFFFFF')
+    color_very_unhealthy: Very Unhealthy (default '#660099')
+
+Requires:
+    python-requests: Python HTTP for Humans
+
+Example:
+```
+air_quality {
+    auth_token = 'demo'
+    location = 'Shanghai'
+    format = 'Shanghai: {aqi} {category}'
+}
+```
+@author beetleman
+@license BSD
+"""
+
+try:
+    # python 3
+    from urllib.error import URLError
+    from urllib.request import urlopen, Request
+    from urllib.parse import urlencode
+except ImportError:
+    # python 2
+    from urllib import urlencode
+    from urllib2 import URLError
+    from urllib2 import urlopen, Request
+import json
+
+AQI = (
+    (0, 50, 'icon_good'),
+    (51, 100, 'icon_moderate'),
+    (101, 150, 'icon_unhealthy_sensitive'),
+    (151, 200, 'icon_unhealthy'),
+    (201, 300, 'icon_very_unhealthy'),
+    (300, None, 'icon_hazardous')
```

Comment:
`500 < None` == `False` so this is no good you could use `int('inf')` (infinity),  `500 < int('inf')`  == `True`

values like `(0, 50, 'good')`, might be easier to use.  But this is looking like a threshold to me so really you don't need that second number at all.  It would also simplify `_key()` slightly.